channel/src/main/java/com/synopsys/integration/alert/channel/jira/common/distribution/delegate/JiraIssueTransitioner.java

```diff
@@ -72,7 +72,7 @@ protected final boolean isTransitionRequired(ExistingIssueDetails<String> existi
             return DONE_STATUS_CATEGORY_KEY.equals(statusCategoryKey);
         } else if (IssueOperation.RESOLVE.equals(transitionType)) {
             // Should resolve?
-            return TODO_STATUS_CATEGORY_KEY.equals(statusCategoryKey);
+            return !DONE_STATUS_CATEGORY_KEY.equals(statusCategoryKey);
```

Comment:
This change will allow Alert to transition issues from an intermediate state to the "Resolve" state.